tests/contrib/futures/test_propagation.py

```diff
@@ -224,15 +262,14 @@ def fn():
 
         # the trace must be completed
         self.assert_span_count(3)
-        self.assert_structure(
-            dict(name="main.thread"),
-            (
-                (
-                    dict(name="executor.thread"),
-                    (dict(name="nested.thread"),),
-                ),
-            ),
-        )
+        spans = self.get_spans()
```

Comment:
NB: Why are we calling `self.get_spans()` instead of `self.get_root_spans()` here (to access the `"main.thread"` span)?

Reply:
🤷🏻 I think just a decision in the moment, since there are only 3 spans I figured it would be simple enough to code this way